docs/en_us/platform_api/source/courses/blocks.rst

```diff
@@ -0,0 +1,347 @@
+.. _Courses API Blocks Resource:
+
+########################################
+Courses API Blocks Resource
+########################################
+
+With the Courses API **Blocks** resource, you can complete the
+following tasks.
+
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _Get a list of the course blocks in a course:
+
+****************************************
+Get a List of Course Blocks in a Course
+****************************************
+
+The endpoint to get a list of course blocks in a course is
+``/api/courses/v1/blocks/?course_id=<course_id>``.
+
+=====================
+Use Case
+=====================
+
+Get a list of course blocks in a specified course. Response results depend on
+the requesting user's access level.
+
+=====================
+Example Request
+=====================
+
+GET /api/courses/v1/blocks/?course_id=<course_id>
+
+GET /api/courses/v1/blocks/?course_id=edX%2FDemoX%2FDemo_Course
+&all_blocks=true&requested_fields=graded,format,student_view_multi_device
+
+=====================
+Parameters
+=====================
+
+This view accepts the following parameters.
+
+* block_counts: to return the aggregate number of blocks (including child
+  blocks) of each type specified in the "block_counts" response value.
+  Example: ``block_counts=video,problem``.
+
+* block_types_filter: Specify the types of blocks to be included in results.
+  Possible values are: ``sequential``, ``vertical``, ``html``, ``problem``,
+  ``video``   and ``discussion``. Example: ``block_types_filter=problem,html``.
+
+* course_id: (string, required) The URL-encoded ID of the course whose block
+  data you are requesting. Example: ``course_id=edX%2FDemoX%2FDemo_Course``.
+
+* depth: (integer or ``all``) Specify how far down in the course blocks
+  hierarchy to traverse down. A value of ``all`` specifies the entire
+  hierarchy. The default value is 0. Example: ``depth=all``.
+
+* requested_fields: (list) Indicates fields to return for each block, in
```

Comment:
Controls the fields returned for each block, 

I like "controls" or "specifies" for parameter explanations.